### DISCUSSION TITLE
When I change the system prompt, none of the slash commands work

### DISCUSSION THREAD

**Discussion by NathanKleekamp** [olimorris/codecompanion.nvim#419] [Q&A] [answered]
When I change the `system_prompt`, as documented in the readme, I lose access to all slash commands:

Here's my entire config:

```lua
local claude_latest = "claude-3-5-sonnet-20241022"

require("codecompanion").setup({
  strategies = {
    chat = {
      adapter = "anthropic",
    },
    inline = {
      adapter = "anthropic",
    },
    agent = {
      adapter = "anthropic",
    },
  },
  adapters = {
    anthropic = function()
      local api_key = os.getenv("ANTHROPIC_API_KEY")
      return require("codecompanion.adapters").extend("anthropic", {
        env = {
          api_key = api_key,
        },
        schema = {
          model = {
            default = claude_latest,
          },
        },
      })
    end,
  },
  opts = {
    system_prompt = "You are Claude",
  }
})
```

My ultimate goal is to use the Claud.ai web app [prompt](https://docs.anthropic.com/en/release-notes/system-prompts#claude-3-5-sonnet), which outputs really high quality responses.

![image](https://github.com/user-attachments/assets/29722a55-9536-4a82-ad7d-6c83f4506685)

Is this expected behavior? Is my config wrong?

**Comment by olimorris** [comment]
I can't understand why that would be the case unless you don't have cmp installed?!

Check the [troubleshooting](https://github.com/olimorris/codecompanion.nvim?tab=readme-ov-file#toolbox-troubleshooting) section and try out the minimal.lua file. If you can recreate it, raise an issue and I can take a look.

**Reply by NathanKleekamp** [reply, marked as answer]
`:checkhealth` comes back fine.

With the minimal.lua settings changing the system prompt works fine, too.

I use commander of completion as my completion engine, have for probably 4 years, and it might be that it conflicts with cmp. Another possibility (among many) is that I use nvim's builtin plugin api instead of a plugin manager.

I may have to just live with this because I'm not really interested in switching away from coc at this point. I'm afraid it'll be too disruptive.

Thanks for your work on this project! It is by far the best one of its kind!
